cmd/neofs-adm/internal/modules/storagecfg/config.go

```diff
@@ -0,0 +1,148 @@
+package storagecfg
+
+const configTemplate = `logger:
+  level: info  # logger level: one of "debug", "info" (default), "warn", "error", "dpanic", "panic", "fatal"
+
+node:
+  wallet:
+    path: {{ .Wallet.Path }}  # path to a NEO wallet; ignored if key is presented
+    address: {{ .Wallet.Account }}  # address of a NEO account in the wallet; ignored if key is presented
+    password: {{ .Wallet.Password }}  # password for a NEO account in the wallet; ignored if key is presented
+  addresses:  # list of addresses announced by Storage node in the Network map
+    - {{ .AnnouncedAddress }}
+  attribute_0: UN-LOCODE:{{ .Attribute.Locode }}
+  attribute_1: Price:{{ .Attribute.Price }}
```

Comment:
neofs-adm didn't ask for `Price` attribute in the dialog. Maybe it is better to avoid price for now, because it may mislead users on guarantee profit, however this `Price` attribute only viable for storage group payments, which are quite rare. 

Or specify default price of `100000` as we do in testnet config.

Reply:
Ok, removed price.